users/jheggemann/jheggemann/Uebung622a.java

```diff
@@ -0,0 +1,58 @@
+package jheggemann;
+
+import java.util.Arrays;
+
+public class Uebung622a {
+    
+    private static int[] arr;
+    private static int max, min, sum = 0;
+    private static double avr;
+    private static int count;
+    public static void main(String args[]){
+        //Arry mit Werten füllen
+        FillArray();
+        //Anzahl der Elemente im Array ermitteln
+        GetCount();
+        //Maximalwert ermitteln
+        GetMax();
+        //Minimalwert ermitteln
+        GetMin();
+        //Array aufsummieren
+        GetSum();
+        //Arraydurchschnitt ermitteln
+        GetAvr();
+        System.out.println(String.format("Anzahl: %d", count));
+        System.out.println(String.format("Average: %.1f", avr));
+        System.out.println(String.format("Maximum: %d", max));
+        System.out.println(String.format("Minimum: %d", min));
+        System.out.println(String.format("Summe: %d", sum));
+    }
+
+    private static void GetCount() {
```

Comment:
In Java gilt die Konvention "camel case" für Funktionen. Bsp: getSomethingDone() {}